### DISCUSSION TITLE
61-64 Naxxramas Proposal

### DISCUSSION THREAD

**Discussion by RestoCC** [chromiecraft/chromiecraft#3742] [Suggestions]
Greetings Chromiecraft,

Many players on this server are concerned that the addition of a level 60 Classic Naxxramas would have a negative impact on the server health. Many players rightfully wish to be fast-tracked to TBC due to the population decline and constant raid logging environment of classic wow phases. Staffs and players alike have shared their opinion on the matter and I think the majority of both parties seem to think that going to Outlands after AQ40 is for the best.

### Proposal
Our proposal is to make use of the timeframe of 61-64 bracket and have a Classic Naxxramas release during this period.

- During this content phase, there is only half of all TBC dungeons available
- There is the release of a new BG: Eye of the Storm
- Obviously new zones and therefore new quests and reputation

Aside from these, there will be very little available and some players may not realize that they are wanting to rush to another phase that may not give them much more than they previously had.

Our proposal is to offer it during this bracket. That way people can get into Outlands at the same timeframe as expected already which will assist with population retention, but can also add some benefits:

- Would just be nice to have a tuned Naxxramas to progress and have fun in between the dungeon running and TBC item farm.
- Would also allow you to put more focus on Obsidian Sanctum and Eye of Eternity fixes during Tier7 and a quicker Ulduar when the time comes.
- Another selling point for new players: TBC content but a level 60 Naxxramas that's tuned could prove valuable for enticing them to a new challenge that hasn't really been seen before
- Content diversity is seldom a bad thing
- Tradeable raid transmog is back! This gives players opportunity to farm rarely seen armour looks and share it between guild members

### I would now like to disclose the driving force behind this post.

A few members of the community (none of whom are staff) have expressed great interest in supporting this concept.

- One particular member has said they are willing to translate all spell damage values, including making a tuning that would be well fit for whatever difficulty would be the aim for the raid. 
- They would be willing to make sure all the loot is both correct in terms of drop rates and either A: Display it in a very easily understood manner for someone to implement or B: Offer to do it themselves.

I have also offered to financially back the people who are starting work on the module.

### Preliminary staff feedback:

- We have staff members that were responsible for the downscaling of Naxxramas to level 60 + the scourge invasion event on the server that inaugurated the concept of slow progression that is used here on Chromiecraft, making the implementation fairly easy in their eyes.
- The structure that was used for Onyxia (the progression module + spell corrections) would allow bringing forth Naxxramas with ease, if it were decided.
- The staff also understand there's some concern among players with the release schedule so it might not be the best timing, it may be perceived as a delay to the schedule and refuse to support it for that reason alone
- Staff have asked for this post to gather community perception to the idea. Staff need to see that a considerable amount of the current player base would like the idea of this Naxxramas

### Conclusion
Regardless of outcome, the players who have offered their time to work on this project have agreed with staff to continue the project for now even if it does not get the greenlight as it may help with future work or Level 80 Naxx. Please offer your feedback below on your thoughts of the solution. 

- Would you be okay with their being a Naxxramas module released during 61-64?
- Are you heavily against this proposal? If so, why?
- Perhaps you dont care either way for this change, would you support it for the sake of others?
- Do you have any concerns or fears? Would you like to see any precautions taken?

Sample:
![Naxx](https://user-images.githubusercontent.com/97218609/177948766-3f26f5fe-2497-4221-802a-625f43c12c75.png)

**Comment by Yurimir** [comment]
60 Naxx, Best Naxx.

**Comment by Xenbiotic** [comment]
So you're saying I can finally fly in outland on my mountless druid AND have Naxx at the same time? Count me in!!! - Awo

**Comment by Spook9** [comment]
Yes, all brackets before 70 will have a content drought sooner or later so having something to do in those times besides AQ/BWL/MC would be neat!

This said it needs to remain independent of the general phasing schedule since pushing back TBC is not gucci.

If this also helps Wrath Naxx testing i only see it as an upside, so win-win-win!

**Comment by Frodhi** [comment]
Resto,
If i could post the gif of Morty's "Count me in" i would.
That idea is actually the best of both worlds.
It fits the bracket cause the gear is on par, if not better than the dungeon ones, so that way, casuals can get decent loot from the 5man stuff, while the more raiding focused guilds can try their luck on the 64 tuned Naxx.

**Comment by elthehablo** [comment]
Good solution as opposed to just throwing Naxx at the end of the vanilla phase! I support this

**Comment by RestoCC** [comment]
Thank you all for your feedback so far! I really appreciate it and im sure staff do too

**Comment by balleny** [comment]
More or less completely against it.

It has literally zero benefit for the project in terms of getting a 3.3.5 wotlk server fixed to a blizzlike state.

> Would also allow you to put more focus on Obsidian Sanctum and Eye of Eternity fixes during Tier7 and a quicker Ulduar when the time comes.

1. it would take away focus from everything that is up and available during the 61-64 phase as people would spend more time in naxx. bored people are the ones who test out shit and find the weird bugs.
2. People might even complain about having to do Naxx again.
3. wotlk classic will make adjustments to naxx, some are already announced, some arent. IMO it would make much much more sense to use the release of wotlk classic to go over a lot of the AzerothCore stuff and bring it on par with wotlk classic as for once, there is a proper data basis available again.
4.  If you want to make it accessible for the sake of making it accessible, why use the 25 ppl implementation instead of the 10 ppl implemention (a bit /s but still...)

**Reply by RestoCC** [reply]
Hey appreciate the feedback.

Are you disagreeing with the point I made about OS/EOE?

Alot of the time with Naxx can and would be assisted by players who have volunteers for the project. Further, i elaborated that most of the world to implement it has already been done thanks to prior knowledge being done originally by the staff members **and** the server having done the same for Onyxia's Lair which was needed for Ony Cloaks.

Sure people may complain about Naxx but then again, others would surely be happy with the idea and you dont _have_ to do Naxx

Your 3rd point I think is relevant to everything this server is about in general, i also think that is partially why CC has moved from a Blizzlike server to a QoL-enhanced server.

I dont quite understand your fourth point if you could elaborate for me please.

Thank you!

**Reply by Wheemix** [reply]
I'm always happy to see the opposite arguments in any discussion so thank you for voicing them.

1. Considering the weekly lockouts for raids, I doubt people would spend enough time in Naxx to significantly impact time spent in Outlands, there will be plenty of rep to grind, quests to complete etc that even people raiding Naxx will want to do. If anything, I could actually foresee a drop in player numbers due to a multiple months long drought in raid content.
2. I usually don't like arguments like "hurr durr if you think RDF makes the game worse just don't use it, make your own group and walk to the dungeon", because that's just shooting yourself in the foot. However in this case I think it is valid to say if you don't want to go Naxx, then just don't. It's not like there is a next raid tier that you need the gear for, it's merely to add optional content/challenges for those that want it.
3. Fair point here, and if this was to be worked on by the actual CC devs I would completely agree, but since there are volunteers to work on and fund this Naxx project, I don't see the downside in letting them do it anyway. (not to mention less bugs in the lvl 80 version).
4. Again, fair point here. Comes down to preference so I have no arguments against it.

My main stance is, that it should not delay the actual development of new content already planned, as others have already expressed above. 

> "It has literally zero benefit for the project in terms of getting a 3.3.5 wotlk server fixed to a blizzlike state."

Besides the enjoyment of the playerbase? Does Transmog have a benefit in terms of getting a blizzlike 3.3.5 state? This project, like any other server, does not work without the players spending their time here.

**Reply by balleny** [reply]
1. considering naxx is not getting implemented, people would still run Aq40 and bwl simply for the items that arent getting replaced once we hit 70. in case naxx is available, people would run naxx instead. More lvl 60 content available, means people spend less time in the 61-64 (or later) bracket. and as i wrote before, it will take away peoples focus from the bracket itself. I am not talking about the devs time investment, but on the users reporting bugs - if people are 25% of their time occupied with Naxx, it means there could be a lot less bug reports for issues from the 61-64 bracket. 
2. I for once joined the server because of AzerothCore, not for ChromieCraft. That is also, why I disagree with most of the custom adjustments that diverted ChromieCraft away from baseline AzerothCore. Be it the availability of Onyxia, Tmog, this new custome (bullshit), reimplementing battleground marks and other decision. Including the idea to implement a not lvl 80 version of Naxxramas. but this is a personal opinion.
3. regarding my last point: originally, Naxx was a 40 player raid. in wotlk, its both a 10 player and 25 player version. The 10 player version is actually the more complex one regarding implementation E.g. 4 horsemen are scripted differently and need additional modifications. Thus, you could argue that if you really want to do Naxx now to reduce implementation effort in the first raid phase of lvl 80, that you would do most of the heavy lifting now and implement a Naxx10 version, not a Naxx25.
4. and lastly, phases not being "overloaded" with content is IMO a good point. people will end up doing stuff more often and potentially deep dive more into things that are getting reported finally (again, the AZ pov, not the CC pov)

**Reply by heyitsbench** [reply]
On point 2 of balleny's reply, I'm of the opinion that custom changes are perfectly fine for this server, as they showcase one of the draws to AC for people that want to run their own server; the modules. Everything that's blizzlike makes its way to AzerothCore, and these modules are not only seeing use and thus having issues be reported, but are also being improved. I moved to AzerothCore solely due to the modules, everything else I was interested in I could've been satisfied by TC. Maybe I'm biased, seeing as I have worked on custom stuff myself for AC, but that's my take on the matter.

**Comment by anhility** [comment]
Personally I'm neutral about this but can see the value of raid content during the level 61-64 bracket, especially if it will be 2-3 months (or more) until Tier 4 is available.
And if a third party can help with down-scaling Naxx so would the other developers be able to focus on the project plan.

### Possibilities:
- Content during a bracket without much else to do raid-wise.
- The Naxx60 gear is also in line with the lvl65 gear you can get in TBC.
- The possibility of getting [Atiesh](https://wotlkdb.com/?item=22630) for casters and Tier 3 as tmog.

### Concerns:
- Does it align with the concept of a 3.3.5 blizzlike server that was the goal for the project?
  - I personally don't mind but I know people who wants a "pure" 3.3.5 experience.
- How much extra time would this module take from the developers working on the lvl 65+ content?
- Would Naxx be tuned to lvl 60 or 64?
  - Pretty much all classes get new major abilities between 60-64 so using dps/heal numbers from lvl 60 might not be viable for raid tuning.
- Will achievements be disabled?
  - I don't see the justification of having achievements enabled for this module when it's during the TBC era.
  - ~~Think it was an oversight with Onyxia60 that people can earn the lvl 80 achievements.~~ Explained by Nyeriah below.
  - Naxx60 don't have any achievements (not even FoS) so a solution would be to just make the achievement function always return false.
    - `CheckAchievementCriteriaMeet()` in the source files.

**Reply by heyitsbench** [reply]
> Will achievements be disabled?

Your solution to this is a little overengineered, all it would take is a disable similar to how I did [here](https://github.com/azerothcore/mod-progression-system/pull/219/files).

**Reply by Nyeriah** [reply]
Achievements in Onyxia were fully intended as the encounter is basically the same as the 80 version and in par with difficulty.

**Reply by anhility** [reply]
> Achievements in Onyxia were fully intended as the encounter is basically the same as the 80 version and in par with difficulty.

I see. Just my personal taste as I would rather see encounter specific achievements during the same phase the content was originally added.

**Reply by Hyperbearr** [reply]
This feels like a very valid concern/point but perhaps one that can be decided by the community via polling once a decision is made as to whether or not to go ahead with introducing level60 naxx?

**Reply by anhility** [reply]
> > Will achievements be disabled?
> 
> Your solution to this is a little overengineered, all it would take is a disable similar to how I did [here](https://github.com/azerothcore/mod-progression-system/pull/219/files).

I admit I've no idea how the game handle checking achievement criteria on the backend.
My suggestion is just a dirty hotfix from what I could easily find when looking though the encounter source files.

**Comment by palimoschnitzel** [comment]
Ye, count me in. Never experienced Classic Naxx, so it'd be fine experience. Also, I hope that Classic Naxx quests will be back too?

**Comment by Pinio1** [comment]
1. No problem for naxx in bracket 61-64 as long as it does not delay tbc release.
2. This NAXX wont be vanilla NAXX which was made for 40players, so do expect exactly same raid in wotkl. Vanila NAX had some awesone mechanics which were removed in wotkl.
3. This should not be challanged raid, do not make any customs nerf / buffs like healing, bosses hp. It is none sense to buff such raid at tbc phase - stay to blizzlike numbers as close as possible

**Comment by k5blazerfl** [comment]
I’m not a fan of this idea at all. For me it’s all or nothing. It’s either finish 60 out with Naxx or leave it for Northrend. This proposal just seems like a bad compromise.

**Reply by RestoCC** [reply]
Hi there would you mind going into some more detail for us? Thank you!

**Reply by k5blazerfl** [reply]
*Why go through the hassle of implementing a hack to enable a raid that we will be seeing again in a later bracket? Especially if it’s only going to be a scaled down 10or25 wrath version and not it’s own unique raid.

*Why regress and try to implement Naxx during a time when the server should be progressing and looking towards “Beta Testing” new content? If people are in Naxx 60 getting T3 they are not out exploring the Outlands. I’d rather see people step up and donate their time for quicker brackets, not spending time creating a stopgap for content stagnation till Kara.

**Reply by CoolFangs** [reply]
"*Why go through the hassle of implementing a hack to enable a raid that we will be seeing again in a later bracket? Especially if it’s only going to be a scaled down 10or25 wrath version and not it’s own unique raid."

Why not? They already did it once with Onyxia.

"*Why regress and try to implement Naxx during a time when the server should be progressing and looking towards “Beta Testing” new content? If people are in Naxx 60 getting T3 they are not out exploring the Outlands. I’d rather see people step up and donate their time for quicker brackets, not spending time creating a stopgap for content stagnation till Kara."

Is Naxx not new content? This is testing that was going to happen at some point regardless. And why do you think people raiding Naxx would prevent them from going to Outland? It's a raid that takes a couple hours a week at most, the rest of the time would be spent exploring and testing the Outland content.

**Reply by balleny** [reply]
Cause
1. onyxia wasnt wanted and planned initially and just got introduced by a single dev. If i recall correctly, it was neither plan of the progression schedule (for lvl 60) nor did the community had a say in it (and could have declined a release)
2. naxx will end up taking away multiple hours (either because its made difficult, or being easy and run on alts, too) and make the dungeon gear and quest completly irrelevant. The amount of people going through the quests/dungeons and finally reporting issues (mind you, not everyone is actually reporting stuff) will be heavily reduced

**Comment by CoolFangs** [comment]
I very much like this idea. I think this is a great way to satisfy both raiders and people patiently waiting for more world content. 

I believe letting raiding die for multiple leveling tiers wouldn't be good for the long term health of the server, many people who are primarily raiders and/or focused on min max progression are going to level up, get whatever new gear they can from Outland dungeons, and then quit until the next set of levels because they really have nothing more to do if they aren't interested in completionism. 

At the same time many people that don't care so much for raiding have already checked out of the server while they wait for new stuff to do, so I understand the desire to not hold off Outland any longer than necessary.  Releasing both as suggested would give the best of both worlds.

-Asami

**Reply by RestoCC** [reply]
Hi Asami thanks for the feedback. 

I agree with your feedback there and hopefully we can convince the staff of the benefit this could bring!

**Comment by lord-spirit** [comment]
**Would you be okay with their being a Naxxramas module released during 61-64?**
I would, yes. I don't think it would split the playerbase too much and going 3 whole phases without new raid content would likely negatively effect the total population count. One question I have is why not do it during the middle phase (65-69)? So, there would be one phase without a raid, one with, one without, and then onto Tier 4. Just a thought if we already have to deal with possibly scaling it to lvl 64.

**Are you heavily against this proposal? If so, why?**
No.

**Perhaps you dont care either way for this change, would you support it for the sake of others?**
I would probably raid Naxx during this phase if it were available, so I support it for myself.

**Do you have any concerns or fears? Would you like to see any precautions taken?**
My main fear is that if it is released in a buggy state, people will focus on it more than the actual content phase: 61-64 and the associated dungeons, which could make that phase last longer as people are doing the *shinier* content. I understand that you're willing to do the up-front development, but how will you manage the long term bug-fixing? There will be bugs and they will make or break this naxx60 implementation.

**Reply by RestoCC** [reply]
Thanks for the feedback!
